mne/tests/test_evoked.py

```diff
@@ -452,3 +452,33 @@ def test_array_epochs():
     types = ['eeg'] * 19
     info = create_info(ch_names, sfreq, types)
     assert_raises(ValueError, EvokedArray, data1, info, tmin=-0.01)
+
+
+def test_add_channels():
+    """Test evoked splitting / re-appending channel types
+    """
+    evoked = read_evokeds(fname, condition=0)
+    evoked.info['buffer_size_sec'] = None
```

Comment:
Hmmm, I just tried creating an EvokedArray, confirmed that "buffer_size_sec" was in there, then wrote to disk and read it back in, and the resulting object no longer had that field. Maybe this would end up being a problem? Either way it sounds like there should be more consistency on the part of this attribute...